### ISSUE TITLE
Allow players to join an existing game

### ISSUE THREAD

**Issue by kingdomlevel** [kingdomlevel/toast#13]
Any user who is not the host:
- should be able to enter a valid game URL and be directed to a Room in lobby state
- **should not** be able to see game admin panel

! Depends on #11

**Comment by kingdomlevel** [comment]
As part of this issue we should also visualise a list of players in the lobby!

**Comment by kingdomlevel** [comment]
This will require:
- linking up the join game button to push `/gameCode` to history
- making a new component `PlayerList` to show the nicknames in current room

**Comment by PorcelainAddict** [comment]
Also should this not include a limitation on players? If I am the host and set it to 6 players, and my amigos amigo tries to join they should get a message saying "Lobby is full."?

**Comment by kingdomlevel** [comment]
> Also should this not include a limitation on players? If I am the host and set it to 6 players, and my amigos amigo tries to join they should get a message saying "Lobby is full."?

I would say this would depend on the admin panel issue #12 - i'm tempted to say that the limit could be incorporated into #12 or be its own issue after?
